static/js/lib/search.js

```diff
@@ -247,3 +248,9 @@ export const buildSearchQuery = ({
   }
   return builder.build()
 }
+
+export const mergeFacetResults = (...args: Array<FacetResult>) => ({
+  buckets: args
+    .map(R.prop("buckets"))
+    .reduce((buckets, acc) => R.unionWith(R.eqBy(R.prop("key")), buckets, acc))
```

Comment:
I think it will work ok without it, but you probably want to add a `[]` here as the starting value for the `.reduce` call

Reply:
Do you mean this: `.reduce((buckets, acc) => R.unionWith(R.eqBy(R.prop("key")), [], buckets, acc))`? That led to some strange behavior so I left it as is.

Reply:
ah sorry no, `.reduce` takes two arguments, a function and a value that will be the value of `acc` the first time the function is called. I was just suggesting adding `[]` as the initial value for clarity, but it's optional so feel free to leave as-is.